waspc/run

```diff
@@ -1,35 +1,41 @@
 #!/usr/bin/env bash
 
-# This script defines common commands used during building / developing
+# This script defines common commands used during building / developing `waspc` project
 # and makes it easy to run them.
 
 SCRIPT_DIR=$(cd `dirname "${BASH_SOURCE[0]}"` && pwd)
 PROJECT_ROOT=$SCRIPT_DIR
-THIS=$0
 COMMAND=${1:-watch}
 shift
 ARGS=("$@") # NOTE: This is a bash array!
 
 BOLD="\033[1m"
-UNDERLINED="\033[4m"
 RESET="\033[0m"
 LIGHT_CYAN="\033[96m"
+YELLOW="\033[33m"
 GREEN="\033[32m"
 RED="\033[31m"
 DEFAULT_COLOR="\033[39m"
 
-BUILD_CMD="cabal build all"
-INSTALL_CMD="${SCRIPT_DIR}/tools/install_packages_to_data_dir.sh && cabal install --overwrite-policy=always"
-BUILD_ALL_CMD="cabal build all --enable-tests --enable-benchmarks"
+WASP_PACKAGES_COMPILE="${SCRIPT_DIR}/tools/install_packages_to_data_dir.sh"
+BUILD_HS_CMD="cabal build all"
+BUILD_HS_FULL_CMD="cabal build all --enable-tests --enable-benchmarks"
+BUILD_ALL_CMD="$WASP_PACKAGES_COMPILE && $BUILD_HS_CMD"
```

Comment:
I reorganized this a bit so now we have `build`(hs), `build:all`, and `build:ts-packages`.